engine/src/main/java/org/destinationsol/assets/Assets.java

```diff
@@ -163,6 +164,22 @@ public static Json getJson(String path) {
         throw new RuntimeException("Json " + path + " not found!");
     }
 
+    /**
+     * Loads a Lang (.lang) from the current environment. Throws an exception if the asset is not found.
+     *
+     * @param path A String specifying the desired asset.
+     * @return The loaded Lang.
+     */
+    public static Lang getLang(String path) {
+        Optional<Lang> langOptional = assetHelper.get(parsePath(path), Lang.class);
+
+        if (langOptional.isPresent()) {
```

Comment:
There is actually a method, `Optional#orElseThrow(Throwable)` that will either return the contained item or, if it's not present, throw the given exception.

That would let you reduce this code down to:
```java
return assetHelper.get(parsePath(path), Lang.class)
            .orElseThrow(new RuntimeException("Lang file " + path + " not found");
```

Reply:
For whatever reason it's stating that that method can't be applied to an Optional. I tried to shuffle it around a bit but I was unsuccessful.  The rest of the methods are structured similarly to what I originally had, but if you'd like I could do some more research and fix it.